librz/core/cmd_descs/cmd_descs.h

```diff
@@ -4,11 +4,9 @@
 // WARNING: This file was auto-generated by cmd_descs_generate.py script. Do not
 // modify it manually. Look at cmd_descs.yaml if you want to update commands.
 //
-
```

Comment:
This change is completely unnecessary, caused by the removal of some empty strings in the python script below